packages/react-dom/src/__tests__/ReactDOMInput-test.js

```diff
@@ -444,7 +444,7 @@ describe('ReactDOMInput', () => {
 
   it('should take `defaultValue` when changing to uncontrolled input', () => {
     const node = ReactDOM.render(
-      <input type="text" value="0" readOnly="true" />,
+      <input type="text" value="0" readOnly={true} />,
```

Comment:
This test was broken by adding the warning on `"true"`. I opted to just fix what the warning was complaining about, which doesn't change the semantics of this test AFAICT.